src/styles/styles.js

```diff
@@ -665,14 +665,6 @@ const styles = {
         marginLeft: 48,
     },
 
-    offlineIndicator: {
```

Comment:
We are still using these styles in OfflineIndicator. I don't think we should remove this.